src/SourceTerm/Manager.cpp

```diff
@@ -123,11 +123,22 @@ void seissol::sourceterm::transformNRFSourceToInternalSource( glm::dvec3 const&
   faultBasis[6] = subfault.normal.x;
   faultBasis[7] = subfault.normal.y;
   faultBasis[8] = subfault.normal.z;
-
-  double mu = (subfault.mu == 0.0) ? material.mu : subfault.mu;
-  pointSources.muA[index] = mu * subfault.area;
-  pointSources.lambdaA[index] = material.lambda * subfault.area;
-
+  
+  pointSources.A[index] = subfault.area;
+  switch(material->getMaterialType()) {
+    case seissol::model::MaterialType::anisotropic:
+      if (subfault.mu != 0) {
+        logWarning(0) << "There are specific fault parameters in the source parameter file. We use anisotropic materials. This is not compatible.";
```

Comment:
Maybe use logError? Or can we recover from this situation?

Reply:
Yes, if the material around the source is isotropic, then the behavior is fine. I've updated the warning text.